testnet/node01-insert-gran.json

```diff
@@ -0,0 +1,10 @@
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "author_insertKey",
+    "params": [
+        "gran",
+        "festival fault employ absorb candy luggage rubber code prepare amateur subject rich",
```

Comment:
So anyone can see the keys and import the keys. What's the worst scenario?

Reply:
These accounts are copied from **account.md**, which are already tracked by git history. A better way is to generate some new accounts and encrypt them (using git-crypt). 

We developers share encryption key.

Reply:
Now the accounts are encrypted.